core/cc/dl_loader.cpp

```diff
@@ -29,23 +29,19 @@
 
 namespace {
 
+// load defs
 #if TARGET_OS == GAPID_OS_WINDOWS
-
-void* load(const char* name) {
+void* load(const char* name, bool must = true) {
```

Comment:
Maybe keep the signature of `load` as it was, let it return `nullptr` and add a target-generic `must_load` ?